scripts/ci/docker-compose/integration-kerberos.yml

```diff
@@ -79,4 +79,11 @@ volumes:
 
 networks:
   example.com:
-    external: true
+    name: example.com
+    driver: bridge
+    attachable: true
```

Comment:
Hmmm @mik-laj this is failing for me on self-hosted runners with the message:

https://github.com/apache/airflow/runs/1831096008?check_suite_focus=true#step:6:3864
```
  Creating Kerberos network
  ./scripts/ci/testing/ci_run_airflow_testing.sh: line 58: kerberos::create_kerberos_network: command not found
  The Compose file '/home/runner/actions-runner/_work/airflow/airflow/scripts/ci/docker-compose/integration-kerberos.yml' is invalid because:
  networks.example.com value Additional properties are not allowed ('attachable' was unexpected)
```

Any ideas? Is it a particular version of docker compose or docker?

Reply:
It looks like attachable was only added in v3.1 of the compose spec, but we are on 2.2

Reply:
What version of docker-compose do we have installed?  Docker Compose 1.28.2 is preinstalled on the hosted runner. https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

Reply:
Installation script: https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/installers/docker-compose.sh

Reply:
Ah, 1.25.0. I'll try updating it.

Reply:
(I'm not sure we need the attachable flag though -- it seems to be the default.) Still, will update ver of compose.

Reply:
https://docs.docker.com/compose/compose-file/compose-file-v2/ only mentions attachable once, but not as a field.

Reply:
I also do not know if we need this flag.  I checked the network configuration that is created by docker-compose using the ``docker inspect example.com`` command and entered the identical configuration in the `docker-compose.yaml` file.  Kerberos is very sensitive to changes in the network, so I didn't want to break anything.

To check the available options in the specification, I chose these specifications in the IDE.
<img width="469" alt="Screenshot 2021-02-04 at 16 47 05" src="https://user-images.githubusercontent.com/12058428/106917553-a7f23600-6708-11eb-86fa-bdc94894c862.png">
Now I can press CTRL + field and see the full specs.
<img width="643" alt="Screenshot 2021-02-04 at 16 48 06" src="https://user-images.githubusercontent.com/12058428/106917666-c6583180-6708-11eb-87cd-b7df201cd210.png">

Reply:
The only published compose spec I was able to find was for v3/latest -- so that might be what that is using.

Well, lets see what my PR that removes it does to the tests.